hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -31,5 +47,18 @@ describe('HuBMAP', () => {
 
     // Footer:
     cy.contains('Supported by the NIH Common Fund');
+
+    cy.get('#loggedin-menu').should('exist');
+
+    // auth cookie should be present
+    cy.getCookie('email').should('exist');
+
+    // UI should reflect this user being logged in
+    cy.get('#loggedinemail').should('contain', 'test@gmail.com').should('exist');
+    cy.get('#loggedin-menu').children().get('#logout-menuitem').should('exist');
+    cy.get('#loggedin-menu').children().get('#logout-menuitem')
+      .children()
+      .get('#logout')
+      .should('exist');
```

Comment:
As discussed, we should avoid path-based tests: The are fragile, and harder to fix if they do break.

Reply:
sorry i am not sure we discussed this. but will see what best i could, given that you are an expert in cypress, i am not. thanks.